sale_discount_display_amount/hooks.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 ACSONE SA/NV
```

Comment:
Why hook is needed for these fields? Shouldn't Odoo already take care of computing the initial values of those fields during installation?

Reply:
I think this is was done because if you have an existing environment with plenty of orders, the compute with ORM is quite long to compute.